### PR TITLE
Arsdorint.GitHub.io

### PR DESCRIPTION
**Pull request by arsdorintbp2003**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/arsdorint.json
Comment (WilliamDavidHarrison) [inline domains/arsdorint.json:8]:
```suggestion
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — CHANGES_REQUESTED** [review]
404

**Comment by arsdorintbp2003** [comment]
I have followed the instruction in this link, https://www.is-a.dev/docs/github_pages/, but when I paste the arsdorint.is-a.dev into the custom domain it is as follow:
image

**Comment by WilliamDavidHarrison** [comment]
> I have followed the instruction in this link, https://www.is-a.dev/docs/github_pages/, but when I paste the arsdorint.is-a.dev into the custom domain it is as follow:
> image

Please wait for this PR to be merged

**Comment by arsdorintbp2003** [comment]
Oh you mean this? And who would review those things? 
image

**Comment by WilliamDavidHarrison** [comment]
> Oh you mean this? And who would review those things?
> image

A maintainer

**Comment by arsdorintbp2003** [comment]
So normally how long does it takes?

**Comment by WilliamDavidHarrison** [comment]
> So normally how long does it takes?

it can take a few days

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website also please fix the merge conflicts

**Review by MaskDuck — DISMISSED** [review]

**Comment by arsdorintbp2003** [comment]
I have resolve the conflict by changing the content in arsdorintbp2003:arsdorint.github.io

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please update your username and verify the records.

Also, where is your website located?

**Comment by WilliamDavidHarrison** [comment]
Newer PR: #4290

**Pull request closed without merging**
